website/docs/r/dashboard.html.markdown

```diff
@@ -0,0 +1,180 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_dashboard"
+sidebar_current: "docs-azurerm-resource-portal-dashboards"
+description: |-
+  Manages a shared dashboard in the Azure Portal.
+---
+
+# azurerm_dashboard
+
+Manages a shared dashboard in the Azure Portal.
+
+It is recommended to follow the steps outlined 
+[here](https://docs.microsoft.com/en-us/azure/azure-portal/azure-portal-dashboards-create-programmatically#fetch-the-json-representation-of-the-dashboard) to create a Dashboard in the Portal and extract the relevant JSON to use in this resource. From the extracted JSON, the content within the `properties: {}` object can used. Variables can be injected as needed - see below example.
```

Comment:
could we move this note below the `dashboard_properties` field below?

Reply:
done